backend/docker-compose.yml

```
@@ -0,0 +1,42 @@
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    networks:
+      - rushticket-network
+
+volumes:
+  rushticket-db:
+
+networks:
+  rushticket-network:
+    driver: bridge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)